noti.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime, timedelta
+import requests
+import googleapiclient.discovery
+
+# Replace with your FCM server key
+FCM_SERVER_KEY = "AIzaSyAtZw82wLJBShSjjsh2fEDJ"

```

Comment:
API keys, server keys should be hidden or masked. Use a .env file and keep it within your team don't push to git.